docs/tutorials/podman_tutorial.md

```diff
@@ -8,114 +8,9 @@ commands with Podman.
 **NOTE**: the code samples are intended to be run as a non-root user, and use `sudo` where
 root escalation is required.
 
```

Comment:
Everything from this file that was not already available in `install.md` has been moved there.